docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,116 @@
+:template: 2018/base.html
+
+Call for proposals
```

Comment:
Ah forgot to ask this, do we want to title-cap or sentence cap? usually H1's get title-caps

Reply:
I'm not picky, so long as we're consistent (sentence case is my default, atm, but if you'd rather title, that's fine by me)